### DISCUSSION TITLE
disable paginated navigation

### DISCUSSION THREAD

**Discussion by GambitFPS** [facebook/docusaurus#6525] [Q&A] [answered]
Is there a way to paginated navigation (previous/next)?

**Comment by slorber** [comment, marked as answer]
frontmatter: 

https://docusaurus.io/docs/sidebar/multiple-sidebars#generating-pagination

```md
---
pagination_next: null
pagination_prev: null
---
```

Please ask easy questions on discord and search the docs first

**Reply by r-colvin** [reply]
please be aware that (as of writing at least) it should be pagination_prev (pagination_previous does not work) - documented here: https://docusaurus.io/docs/2.0.0-rc.1/sidebar/multiple-sidebars#generating-pagination

<img width="840" alt="image" src="https://user-images.githubusercontent.com/18327771/187849151-bfe11377-4569-4703-b534-617fa7a6c123.png">

**Reply by slorber** [reply]
thanks, edited

**Comment by doroncy** [comment]
how can i disable pagination for the whole site? not on a specific doc ?

**Reply by slorber** [reply]
You can swizzle DocPaginator + BlogPostPaginator and render null. We don't offer any first-class API for this currently

**Reply by johnnyreilly** [reply]
Possibly related discussion: https://github.com/facebook/docusaurus/discussions/8311

**Reply by slorber** [reply]
Here @doroncy  seems more interested in Docs paginated navigation:

<img width="1022" alt="CleanShot 2022-11-18 at 11 29 23@2x" src="https://user-images.githubusercontent.com/749374/202681197-0b3f303e-b5f5-489f-ad7b-65e25f39cd24.png">

> how can i disable pagination for the whole site? not on a specific doc ?

It's not clear what "whole site" means, and if this includes disabling blog/blog tags/docs tags paginated lists

**Reply by doroncy** [reply]
i do not use blogs in the site, and only pages / docs and i would like to disable the pagination from all docs and not add 
```
---
pagination_next: null
pagination_prev: null
---
```
on every single doc page.

**Reply by tomy0000000** [reply]
@doroncy I'm having the same issue now. Did you manage to solve this without adding `pagination_next: null` and `pagination_prev: null` to all pages?

**Reply by slorber** [reply]
@tomy0000000 this upcoming feature should make it easy to solve this problem globally without adding the frontmatter everywhere: https://github.com/facebook/docusaurus/issues/5568

I'd like to include it in the first v3 release

**Reply by tomy0000000** [reply]
@slorber Thanks for the info. I'm quite surprised that the solution is not altering some key inside `docusaurus.config.js`, but I'm glad we have a proposal on track now. Is there a roadmap or timeline for v3? I can't find it anywhere.

**Comment by discolt** [comment]
custom.css

.pagination-nav {
  display: none;
}
